### ISSUE TITLE
`gts check` leads to `TypeError: Cannot read property 'getFullText' of undefined`

### ISSUE THREAD

**Issue by JustinBeckwith** [google/ts-style#86]
```
beckwith-macbookpro:google-api-nodejs-client beckwith$ ls
AUTHORS             COPYING             apis                lib                 samples             test                yarn.lock
CHANGELOG.md        MIGRATING.md        coverage            node_modules        scripts             tsconfig.json
CONTRIBUTORS        README.md           jsdoc-conf.json     package.json        templates           tsconfig.tools.json
beckwith-macbookpro:google-api-nodejs-client beckwith$ gts check lib/*.ts
TypeError: Cannot read property 'getFullText' of undefined
    at /Users/beckwith/.nvm/versions/node/v8.9.1/lib/node_modules/gts/build/src/lint.js:35:55
    at Array.forEach (<anonymous>)
    at lint (/Users/beckwith/.nvm/versions/node/v8.9.1/lib/node_modules/gts/build/src/lint.js:34:14)
    at /Users/beckwith/.nvm/versions/node/v8.9.1/lib/node_modules/gts/build/src/cli.js:77:46
    at step (/Users/beckwith/.nvm/versions/node/v8.9.1/lib/node_modules/gts/build/src/cli.js:33:23)
    at Object.next (/Users/beckwith/.nvm/versions/node/v8.9.1/lib/node_modules/gts/build/src/cli.js:14:53)
    at /Users/beckwith/.nvm/versions/node/v8.9.1/lib/node_modules/gts/build/src/cli.js:8:71
    at new Promise (<anonymous>)
    at __awaiter (/Users/beckwith/.nvm/versions/node/v8.9.1/lib/node_modules/gts/build/src/cli.js:4:12)
    at run (/Users/beckwith/.nvm/versions/node/v8.9.1/lib/node_modules/gts/build/src/cli.js:52:12)
```

**Comment by jinwoo** [comment]
Does it happen only when you explicitly pass the files?

**Comment by JustinBeckwith** [comment]
It seems to only happen when I pass arguments in the folder/*.ext format.  Running `gts check lib/discovery.ts` works fine.  

Same bug applies to `fix`.

**Comment by jinwoo** [comment]
Are you on windows?

**Comment by JustinBeckwith** [comment]
lol no

**Comment by jinwoo** [comment]
This sounds like the wildcard expansion doesn't happen from the shell. Let me take a look.

**Comment by jinwoo** [comment]
Now I know why this happens. It's because the shell expansion for `lib/*.ts` also includes those `*.d.ts` files.

**Comment by jinwoo** [comment]
I'm not sure what's the correct fix here. We could change so that
1. `gts` ignores `*.d.ts` files
2. Discourage people from using wildcards

Maybe 1 is better?

**Comment by JustinBeckwith** [comment]
Interesting!  I'm pretty sure I've used tslint against d.ts files - why wouldn't that work?

**Comment by jinwoo** [comment]
I'm not sure. In google-api-nodejs-client, the `*.d.ts` files are not recognized by the typescript server. My fix will be "Ignore files that are not recognized by the typescript server".

**Comment by JustinBeckwith** [comment]
Fixed by #87
